setup.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+###########################
+# This code block is a HACK (!), but is necessary to avoid code duplication. Do NOT alter these lines.
+import importlib.util
+import os
+
+from setuptools import setup
+
+filepath = os.path.abspath(os.path.dirname(__file__))
+filepath_import = os.path.join(filepath, "..", "core", "src", "autogluon", "core", "_setup_utils.py")
+spec = importlib.util.spec_from_file_location("ag_min_dependencies", filepath_import)
+ag = importlib.util.module_from_spec(spec)
+# Identical to `from autogluon.core import _setup_utils as ag`, but works without `autogluon.core` being installed.
+spec.loader.exec_module(ag)
+###########################
+
+version = "0.1"
+version = ag.update_version(version, use_file_if_exists=False, create_file=True)
+
+submodule = "cloud"
+install_requires = [
+    # version ranges added in ag.get_dependency_version_ranges()
+    "autogluon.common<0.7",
```

Comment:
Are you sure we should have an open version range for the lower end here?

Reply:
+1, we shouldn't cap this unless there is a strong reason for it

Reply:
OK. I think we need a lower end here: `>=0.6`, but probably not a upper cap